test/fluree/server/integration/policy_test.clj

```diff
@@ -146,13 +146,26 @@
                      (-> query-res :body json/read-value first (get "f:assert")))
                   "policy opts prevented seeing bob's secret"))))
         (testing "JWS requests"
+          (let [txn-req {"@context" ["https://ns.flur.ee" test-system/default-context]
```

Comment:
Could you add more commentary to these tests with more `testing` macro calls?

What is this setup code before the test assertion doing? Why should this transaction be authorized? Why should the following one not be?

Reply:
This whole deftest block sets up a simple graph which allows users to view and update their own `ex:secret` predicates. The jws block of tests verifies that using the identity can be correctly verified and used for policy enforcement via the jws mechanism of signing requests.

Before, there were only two cases: that Alice couldn't modify Bob's secret and that Bob's secrets don't appear in Alice's query results. I added the test case for Alice being able to modify her own secret.